WireAPI/Tests/WireAPITests/APIs/BackendInfoAPI/BackendInfoAPITests.swift

```diff
@@ -16,47 +16,48 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-@testable import WireAPI
+import SnapshotTesting
 import XCTest
 
+@testable import WireAPI
+
 final class BackendInfoAPITests: XCTestCase {
 
-    private var httpClient: HTTPClient!
+    private var apiSnapshotHelper: APISnapshotHelper<BackendInfoAPI>!
 
-    // MARK: - V0
+    // MARK: - Setup
 
-    func testGetBackendInfoRequestV0() async throws {
-        // Given
-        let httpClient = HTTPClientMock()
-        let sut = BackendInfoAPIV0(httpClient: httpClient)
+    override func setUp() {
+        super.setUp()
+        apiSnapshotHelper = APISnapshotHelper { httpClient, _ in
+            let builder = BackendInfoAPIBuilder(httpClient: httpClient)
+            return builder.makeAPI()
+        }
+    }
 
-        // When
-        _ = try? await sut.getBackendInfo()
+    override func tearDown() {
+        apiSnapshotHelper = nil
+        super.tearDown()
+    }
 
-        // Then
-        XCTAssertEqual(
-            httpClient.receivedRequest,
-            HTTPRequest(
-                path: "/api-version",
-                method: .get
-            )
-        )
+    // MARK: - Request generation
+
+    func testGetBackendInfoRequest() async throws {
+        try await apiSnapshotHelper.verifyRequest(for: [.v0]) { sut in
```

Comment:
Hmm I think it's bit strange to have the unversioned implementation run with `v0` and also the snapshot contains it in the name `testGetBackendInfoRequest.v0.txt`.

We could add a func to `APISnapshotHelper` that works unversioned or few cases just use `HTTPRequestSnapshotHelper` directly and turn `resourceName: String` optional as the snapshot testing library would accept it optional.

Reply:
done 7f01739c88216e2f37fe33d3d48ed6eace8c3593